ckanext/switzerland/plugins.py

```diff
@@ -137,6 +138,43 @@ def get_helpers(self):
             'ogdch_template_helper_get_active_class': bh.ogdch_template_helper_get_active_class, # noqa
         }
 
+    # IRouter
+
+    def before_map(self, map):
+        """create perma-link route"""
```

Comment:
this is a very fussy comment, sorry, but i think that should be `# create perma-link route` :)